build-info-extractor/src/main/java/org/jfrog/build/extractor/packageManager/PackageManagerUtils.java

```diff
@@ -137,6 +138,12 @@ public static boolean containsSuspectedSecrets(String value) {
                 containsSuspectedSecret(value, accessTokenSecretPrefix, accessTokenSecretMinimalLength);
     }
 
+    public static boolean isJfrogInternalKey(String key) {
+        return key.startsWith(BuildInfoConfigProperties.PROP_PROPS_FILE) ||
+                key.startsWith(BuildInfoConfigProperties.PROP_PROPS_FILE_KEY) ||
+                key.startsWith(BuildInfoConfigProperties.PROP_PROPS_FILE_KEY_IV);
```

Comment:
Let's use StringUtils.contains for the following reasons:
1. It is null safe
2. Sometimes we use variables with prefixes
```suggestion
        return StringUtils.contains(key, BuildInfoConfigProperties.PROP_PROPS_FILE) ||
                StringUtils.contains(key, BuildInfoConfigProperties.PROP_PROPS_FILE_KEY) ||
                StringUtils.contains(key, BuildInfoConfigProperties.PROP_PROPS_FILE_KEY_IV);
```